### PR TITLE
remove obsolete NEWMANGLE

### PR DESCRIPTION
**Pull request by WalterBright**

### CODE REVIEW

File: src/backend/cppman.c [hunk 1]
```diff
@@ -17,7 +17,7 @@
 #include        <string.h>
 #include        "cc.h"
 
-#if !NEWMANGLE
+#if 0
```
Comment (yebblies) [inline src/backend/cppman.c]: Could it be deleted instead?

File: src/backend/newman.c [hunk 2]
```diff
@@ -1219,7 +1213,6 @@ STATIC void cpp_argument_list(type *t, int flag)
         {
             if (i == mangle.argi)               // no match
             {
-#if OVERLOAD_CV_PARAM
```
Comment (yebblies) [inline src/backend/newman.c]: Can the definition be deleted now?

Comment (WalterBright) [inline src/backend/newman.c]: I thought it already was. It's gone from my end.

Comment (yebblies) [inline src/backend/newman.c]: Apparently in master the definition still exists (https://github.com/D-Programming-Language/dmd/blob/master/src/backend/cdef.h#L264) but the two uses in this file are already gone.  Maybe you rebased against an old master?

File: src/backend/cppman.c (hunk 1 above)
Comment (WalterBright) [inline src/backend/cppman.c]: yeah, just removed the file.

**Comment by yebblies** [comment]
Awesome.

**Comment by yebblies** [comment]
Auto-merge toggled on

**Pull request merged**